examples/medplum-patient-intake-demo/src/bots/core/intake-form.ts

```diff
@@ -72,8 +83,75 @@ export async function handler(event: BotEvent<QuestionnaireResponse>, medplum: M
     patient,
     observationCodeMapping.educationLevel,
     observationCategoryMapping.sdoh,
+    'valueCodeableConcept',
     answers['education-level'].valueCoding
   );
 
+  await upsertObservation(
+    medplum,
+    patient,
+    observationCodeMapping.pregnancyStatus,
+    observationCategoryMapping.socialHistory,
+    'valueCodeableConcept',
+    answers['pregnancy-status'].valueCoding
+  );
+
+  await upsertObservation(
+    medplum,
+    patient,
+    observationCodeMapping.estimatedDeliveryDate,
+    observationCategoryMapping.socialHistory,
+    'valueDateTime',
+    { valueDateTime: convertDateToDateTime(answers['estimated-delivery-date'].valueDate) }
+  );
+
+  const insuranceProviders = getGroupRepeatedAnswers(questionnaire, response, 'coverage-information');
+
+  for (const provider of insuranceProviders) {
+    await addCoverage(medplum, patient, provider);
+  }
+
+  // Handle consents
+
+  await addConsent(
+    medplum,
+    patient,
+    !!answers['consent-for-treatment-signature']?.valueBoolean,
+    consentScopeMapping.treatment,
+    consentCategoryMapping.med,
+    undefined,
+    convertDateToDateTime(answers['consent-for-treatment-date'].valueDate)
+  );
+
+  await addConsent(
+    medplum,
+    patient,
+    !!answers['agreement-to-pay-for-treatment-help']?.valueBoolean,
+    consentScopeMapping.treatment,
+    consentCategoryMapping.pay,
+    consentPolicyRuleMapping.hipaaSelfPay,
+    convertDateToDateTime(answers['agreement-to-pay-for-treatment-date'].valueDate)
+  );
+
+  await addConsent(
+    medplum,
+    patient,
+    !!answers['notice-of-privacy-practices-signature']?.valueBoolean,
+    consentScopeMapping.patientPrivacy,
+    consentCategoryMapping.nopp,
+    consentPolicyRuleMapping.hipaaNpp,
+    convertDateToDateTime(answers['notice-of-privacy-practices-date'].valueDate)
+  );
+
+  await addConsent(
+    medplum,
+    patient,
+    !!answers['acknowledgement-for-advance-directives-signature']?.valueBoolean,
+    consentScopeMapping.adr,
+    consentCategoryMapping.acd,
+    undefined,
```

Comment:
We're just going to have to make up a code for this. can we just make the system `http://medplum.com` and the code `BasicADR`